packages/ocs/dashboards/object-service/details-card/details-card.tsx

```diff
@@ -136,6 +161,19 @@ export const ObjectServiceDetailsCard: React.FC<{}> = () => {
           >
             {serviceVersion}
           </DetailItem>
+          <DetailItem
+            key="encryption"
+            title={t('Encryption')}
+            isLoading={!ocsLoaded}
+            error={ocsError}
+          >
+            <EncryptionPopover
+              encryptionDetails={encryptionDetails}
+              hasStorageClassEncryption={hasStorageClassEncryption}
```

Comment:
u don't have StorageClass encryption for object...
```suggestion
              hasStorageClassEncryption={false}
```